Code/Framework/AzToolsFramework/Platform/Linux/AzToolsFramework/API/PythonLoader_Linux.cpp

```diff
@@ -30,12 +32,50 @@ namespace AzToolsFramework::EmbeddedPython
         }
     }
 
-    PythonLoader::~PythonLoader()
+    void PythonLoader::UnloadRequiredModules()
     {
         if (m_embeddedLibPythonHandle)
         {
             dlclose(m_embeddedLibPythonHandle);
         }
     }
-    
+
+    AZ::IO::FixedMaxPath PythonLoader::GetDefault3rdPartyPath(bool createOnDemand)
+    {
+        AZ::IO::FixedMaxPath thirdPartyEnvPathPath;
+        static constexpr const char* thirdPartySubpath = ".o3de/3rdParty";
+        const char* envValue = nullptr;
+
+        // First check if `LY_3RDPARTY_PATH` is explicitly set in the environment
+        if ((envValue=std::getenv("LY_3RDPARTY_PATH")) != nullptr)
```

Comment:
This should use the AZ::Utils::GetEnv function which is portable across all platforms.

Here is an example of using the API from an AZ Core Unit Test: https://github.com/o3de/o3de/blob/development/Code/Framework/AzCore/Tests/Platform/Common/UnixLike/Tests/UtilsTests_UnixLike.cpp#L65-L69